src/main/kotlin/org/jitsi/nlj/MediaSourceDesc.kt

```diff
@@ -97,14 +99,14 @@ class MediaSourceDesc
      * @return the last "stable" bitrate (bps) of the encoding with a non-zero rate
      * at or below the specified index.
      */
-    fun getBitrateBps(nowMs: Long, idx: Int): Long {
+    fun getBitrateBps(nowMs: Long, idx: Int): Bandwidth {
```

Comment:
If this (and related) functions now return a `Bandwidth`, they shouldn't have "Bps" in their names.